action_plugins/tasmota.py

```diff
@@ -10,6 +10,9 @@
 import sys
 import copy
 
+from urllib3.util import Retry
```

Comment:
Do we need to update the [requirements.txt](https://github.com/tobias-richter/ansible-tasmota/blob/master/requirements.txt)?

Reply:
ööööh interesting question, I assume yeah.